### PR TITLE
sample shortcut addin with taskpane and CF

### PR DESCRIPTION
**Pull request by ylu0826**

### CODE REVIEW

File: addins/shortcut-sample-with-taskpane-CF/home.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta http-equiv="Expires" content="0" />
+    <title>Shortcut Sample</title>
+    <script src="https://appsforoffice.edog.officeapps.live.com/lib/beta/hosted/office.js" type="text/javascript"></script>
```
Comment (ghost) [inline addins/shortcut-sample-with-taskpane-CF/home.html:8]: I think we don't need this line if we use the bundle.js.

File: addins/shortcut-sample-with-taskpane-CF/manifest.xml [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+  <Version>1.0.0.0</Version>
+  <ProviderName>Microsoft</ProviderName>
+  <DefaultLocale>en-US</DefaultLocale>
+
+  <!-- The display name of your add-in. Used on the store and various places of the Office UI such as the add-ins dialog. -->
+  <DisplayName DefaultValue="Shortcut Sample with taskpane and CF" />
+  <Description DefaultValue="Shortcut Sample with taskpane and CF" />
+
+  <!-- Icon for your add-in. Used on installation screens and the add-ins dialog. -->
+  <IconUrl DefaultValue="https://officedev.github.io/testing-assets/addins/shortcut-sample/images/button32x32.png" />
```
Comment (ghost) [inline addins/shortcut-sample-with-taskpane-CF/manifest.xml]: We could use the shared one. `https://officedev.github.io/testing-assets/addins/images/button32x32.png`. And elsewhere.

**Review by hjkim-MSFT — CHANGES_REQUESTED** [review]

File: addins/shortcut-sample-with-taskpane-CF/manifest.xml (hunk 2 above)
Comment (ylu0826) [reply addins/shortcut-sample-with-taskpane-CF/manifest.xml]: Fixed

File: addins/shortcut-sample-with-taskpane-CF/home.html (hunk 1 above)
Comment (ylu0826) [reply addins/shortcut-sample-with-taskpane-CF/home.html:8]: removed the bundle.js.

**Pull request merged**